IOHexperimenter/src/IOH_function.py

```diff
@@ -12,27 +12,35 @@
 from collections.abc import Iterable
 import operator
 
-bbob_fid_dict = {1:IOH.Sphere, 2:IOH.Ellipsoid, 3:IOH.Rastrigin, 4:IOH.Bueche_Rastrigin, 5:IOH.Linear_Slope, 
-                6:IOH.Attractive_Sector, 7:IOH.Step_Ellipsoid, 8:IOH.Rosenbrock, 9:IOH.Rosenbrock_Rotated, 10:IOH.Ellipsoid_Rotated, 
-                11:IOH.Discus, 12:IOH.Bent_Cigar, 13:IOH.Sharp_Ridge, 14:IOH.Different_Powers, 15:IOH.Rastrigin_Rotated, 
-                16:IOH.Weierstrass, 17:IOH.Schaffers10, 18:IOH.Schaffers1000, 19:IOH.Griewank_RosenBrock, 20:IOH.Schwefel, 
-                21:IOH.Gallagher101, 22:IOH.Gallagher21, 23:IOH.Katsuura, 24:IOH.Lunacek_Bi_Rastrigin}
-
-pbo_fid_dict = {1:IOH.OneMax, 2:IOH.LeadingOnes, 3:IOH.Linear, 4:IOH.OneMax_Dummy1, 5:IOH.OneMax_Dummy2, 
-                6:IOH.OneMax_Neutrality, 7:IOH.OneMax_Epistasis, 8:IOH.OneMax_Ruggedness1, 9:IOH.OneMax_Ruggedness2, 10:IOH.OneMax_Ruggedness3, 
-                11:IOH.LeadingOnes_Dummy1, 12:IOH.LeadingOnes_Dummy2, 13:IOH.LeadingOnes_Neutrality, 14:IOH.LeadingOnes_Epistasis, 
-                15:IOH.LeadingOnes_Ruggedness1, 16:IOH.LeadingOnes_Ruggedness2, 17:IOH.LeadingOnes_Ruggedness3, 
-                18:IOH.LABS, 19:IOH.MIS, 20:IOH.Ising_Ring, 21:IOH.Ising_Torus, 22:IOH.Ising_Triangular, 23:IOH.NQueens}
+bbob_fid_dict = {1: IOH.Sphere, 2: IOH.Ellipsoid, 3: IOH.Rastrigin, 4: IOH.Bueche_Rastrigin, 5: IOH.Linear_Slope,
+                 6: IOH.Attractive_Sector, 7: IOH.Step_Ellipsoid, 8: IOH.Rosenbrock, 9: IOH.Rosenbrock_Rotated,
+                 10: IOH.Ellipsoid_Rotated,
+                 11: IOH.Discus, 12: IOH.Bent_Cigar, 13: IOH.Sharp_Ridge, 14: IOH.Different_Powers,
+                 15: IOH.Rastrigin_Rotated,
+                 16: IOH.Weierstrass, 17: IOH.Schaffers10, 18: IOH.Schaffers1000, 19: IOH.Griewank_RosenBrock,
+                 20: IOH.Schwefel,
+                 21: IOH.Gallagher101, 22: IOH.Gallagher21, 23: IOH.Katsuura, 24: IOH.Lunacek_Bi_Rastrigin}
+
+pbo_fid_dict = {1: IOH.OneMax, 2: IOH.LeadingOnes, 3: IOH.Linear, 4: IOH.OneMax_Dummy1, 5: IOH.OneMax_Dummy2,
+                6: IOH.OneMax_Neutrality, 7: IOH.OneMax_Epistasis, 8: IOH.OneMax_Ruggedness1, 9: IOH.OneMax_Ruggedness2,
+                10: IOH.OneMax_Ruggedness3,
+                11: IOH.LeadingOnes_Dummy1, 12: IOH.LeadingOnes_Dummy2, 13: IOH.LeadingOnes_Neutrality,
+                14: IOH.LeadingOnes_Epistasis,
+                15: IOH.LeadingOnes_Ruggedness1, 16: IOH.LeadingOnes_Ruggedness2, 17: IOH.LeadingOnes_Ruggedness3,
+                18: IOH.LABS, 19: IOH.MIS, 20: IOH.Ising_Ring, 21: IOH.Ising_Torus, 22: IOH.Ising_Triangular,
+                23: IOH.NQueens}
```

Comment:
I guess Furong should also make function 24 and 25 available..  But, it does not matter now. We could update the package later.